lib/backup_helper.py

```diff
@@ -32,4 +32,6 @@ def recover(file_path):
             if directory not in tar.getnames():
                 raise Exception("Archive is wrong.")
 
+        tar.extractall(path=getenv('HOME', '/home/pi'))
```

Comment:
@rusko124 this would break when running on Balena, wouldn't it?

Reply:
@vpetersson  In Balena cloud we use the environment variable `HOME=/data`, so this should work.

Reply:
Great - we should probably replace that with SRLY_STORAGE or something but fine for now.